tests/test_analytics.py

```diff
@@ -25,7 +25,7 @@ def test_get_order_payloads(order_with_lines):
     for i, line in enumerate(order):
         item = data[i + 1]
         assert item["ti"] == order.pk
-        assert item["in"] == line.product_name
+        assert item["in"] == line.variant.display_product()
```

Comment:
Shouldn't be `str(line.variant)` here?

Reply:
And what about rest of calls of `.display_product()`? As @maarcingebala  said, Do we need it at all? Maybe we can remove it.

Reply:
1. No, because str(line.variant) is `variant.name` or `variant.sku` and `display_product` is `product.name (variant.name)`. We need both names in analytics.
2. I think we shouldn't remove it because google merchant uses it.